packages/editor/src/client/blocks/richTextBlock/toolbar/linkMenu.tsx

```diff
@@ -51,7 +95,21 @@ export function LinkMenu() {
       <Form fluid>
         <FormGroup>
           <ControlLabel>{t('blocks.richText.link')}</ControlLabel>
-          <FormControl value={url} onChange={url => setURL(url)} />
+          <InputGroup>
+            <select
+              style={{backgroundColor: 'white', border: 'none', boxShadow: 'none'}}
+              value={prefix}
+              onChange={e => {
+                setPrefix(e.target.value !== 'other' ? e.target.value : prefixType.other)
```

Comment:
why do you need this check?

Reply:
Actually good point. I originally had it to put in a blank string, but now its doing nothing.